src/components/common/CourseReviewBox.tsx

```diff
@@ -70,6 +79,7 @@ const getProfIndex = (review: any, profsTeaching: any) =>
 
 type CourseReviewBoxContentProps = CourseReviewBoxProps & {
   profsTeachingByCourseId: any;
+  allProfs: any[];
```

Comment:
rip I wish I made types for these, oh well

Reply:
say less

Reply:
no don't do it lol

Reply:
too late